### PR TITLE
B 18188 int removed deprecated personally procured moves model and devseed data

### PR DESCRIPTION
**Pull request by joeydoyecaci**
B-18448

Summary
Removed PersonallyProcuredMoves model from MyMove Due to the extensive modifications to the tests and active code needed to fully remove references to this model, this component will require extensive testing

##Testing Steps

Verify all related Unit tests pass (Changes were all Go, but could indirectly impact JS).
Verify the ability to create PPMs as a customer
Approve/Reject/modify it as a Service Counselor
When signing the shipment as Customer verify that all fields are being populated, and no errors occur

### CODE REVIEW

File: swagger-def/internal.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-swagger: '2.0'
+swagger: "2.0"
```
Comment (pambecker) [inline swagger-def/internal.yaml:1]: just curious why the change from single quotes to double quotes in this file?

File: cmd/milmove-tasks/send_payment_reminder_email.go [hunk 2]
```diff
@@ -1,102 +0,0 @@
...
-	// Email
-	cli.InitEmailFlags(flag)
-
-	// Don't sort flags
-	flag.SortFlags = false
-}
-
-// Command (test eamil): go run ./cmd/milmove-tasks send-payment-reminder
-// Command (send email): go run ./cmd/milmove-tasks send-payment-reminder --email-backend=ses --aws-ses-domain=devlocal.dp3.us --aws-ses-region=us-gov-west-1
-func sendPaymentReminder(cmd *cobra.Command, args []string) error {
```
Comment (pambecker) [inline cmd/milmove-tasks/send_payment_reminder_email.go:48]: Just trying to learn, we don't need this email anymore?  is it only used for PersonallyProcuredMoves old code?

File: swagger-def/internal.yaml (hunk 1 above)
Comment (joeydoyecaci) [reply swagger-def/internal.yaml:1]: This was automatically done by the spectral linter being used to reduce the warnings coming back in CircleCI

File: cmd/milmove-tasks/send_payment_reminder_email.go (hunk 2 above)
Comment (joeydoyecaci) [reply cmd/milmove-tasks/send_payment_reminder_email.go:48]: I believe it is. It was heavily written around the use of that model. It would've been difficult to reproduce with PPM_shipment, nor would it work with PersonallyProcuredMoves omitted.

**Review by paulstonebraker — APPROVED** [review]
Did a full run through the application, encountered no issues or odd behavior. No tests failed on my local. Twice over of code looks good 🤞

**Review by pambecker — APPROVED** [review]
